nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -471,7 +465,7 @@ After going through this checklist, you're ready to go!
 /obj/structure/overmap/fighter/slowprocess()
 	. = ..()
 	use_fuel()
-	if(canopy_breached || (canopy_open && pilot)) //Leak.
+	if(canopy_breached) //Leak air if the canopy is breached.

```

Comment:
I think the ideal solution would be to average the air if the canopy was breached or open on a non-overmap Z, and just remove it on the overmap Z because no atmos. Agree that it didn't make sense to have air behavior depend on the presence of a pilot though.